api/environments/identities/models.py

```diff
@@ -107,9 +107,8 @@ def get_all_feature_states(self, traits: typing.List[Trait] = None):
                 current_flag = identity_flags[flag.feature_id]
                 if flag > current_flag:
                     identity_flags[flag.feature_id] = flag
-
-        if self.environment.project.hide_disabled_flags:
-            # filter out any flags that are disabled if configured on the project
+        if self.environment.get_hide_disabled_flags() is True:
+            # filter out any flags that are disabled
```

Comment:
Oh wow, that's a ticket from a previous project management solution - Clubhouse! We won't have access to that anymore so we should probably just remove this comment. It would be harder to achieve in the database now anyway.